spec/system/posts_index_spec.rb

```diff
@@ -12,43 +12,43 @@
     @posts = Post.all
   end
 
-  it 'has users posts' do
+  it 'should have the name of the author of the post' do
     visit "/users/#{@tom.id}/posts"
     expect(page).to have_content(@tom.name.to_s)
   end
 
-  it 'description' do
+  it 'should have the image of the author or the post' do
     visit "/users/#{@tom.id}/posts"
     expect(page).to have_selector("img[src='https://placehold.co/200x133']")
   end
 
-  it 'has users posts' do
+  it 'should redirect to a particular post' do
     visit "/users/#{@tom.id}/posts"
     click_link 'show post'
     expect(page).to have_content(@first_post.title.to_s)
   end
 
-  it 'description' do
+  it 'should have the number of post of the current user' do
```

Comment:
- [ ] All integration tests are failing on my end. See ScreenShot below for the failing tests and try to fix the test cases.
<img width="778" alt="Screenshot 2023-10-07 at 10 13 31 AM" src="https://github.com/BenM4k/BlogApp/assets/44798044/b09f2f6d-d0cd-41de-9d3b-12d9978237e0">